invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -36,20 +36,20 @@ export default class DocumentMetadata extends Component {
   render() {
     const document = this.props.documentDetails;
     const rows = [
-      { name: 'Title', value: document.title },
-      { name: 'Authors', value: document.authors },
+      { name: 'Title', value: document.metadata.title },
+      { name: 'Authors', value: document.metadata.authors },
     ];
     if (!_isEmpty(document.keywords)) {
```

Comment:
This also needs to use `document.metadata.keywords`.

Reply:
👀